terraform/modules/prom-ec2/paas-config/prometheus.conf.tpl

```diff
@@ -69,3 +69,163 @@ scrape_configs:
     scheme: https
     static_configs:
       - targets: ['canary.verify-main.london.verify.govsvc.uk']
+  - job_name: paas_elasticsearch_for_dm
+    scheme: https
+    basic_auth:
+      username: digitalmarketplace
+      password: ${dm_elasticsearch_metrics_password}
+    metrics_path: '/federate'
+    params:
+      "match[]":
+      - "{job='aiven'}"
```

Comment:
Rather than grabbing all aiven metrics and filtering with `metric_relabel_configs`, could we ask the federate endpoint to only provide the metrics we need?  For example:
```suggestion
      - "disk_free{job='aiven'}"
      - "disk_used_percent{job='aiven'}"
      - "{__name__=~"system_load.*",job='aiven'}"
      - "{__name__=~"elasticsearch_clusterstats.*",job='aiven'}"
```

This will make less work for prometheus since it doesn't have to filter out a bunch of metrics on every scrape.

My only concern is that under the hood, this will construct a giant query string for the `/federate` request, and it may end up with a query that's too large.  I think my answer to that would be multiple scrape_configs to scrape different sets of metrics.

Reply:
This is a good idea but I'd like to avoid changing it for now. This exporter is probably very temporary and in future we won't need to filter out metrics in this way.